.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,53 @@
+## Description
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format (described below)?
+- [ ] Have you added the unit tests to prevent issue reoccurance( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
```

Comment:
I would use impersonal wording like `Has tests...` or `There are tests...`. Not only here, in general.

Reply:
I'll keep that in mind.